packages/manager/modules/sms/src/sms/dashboard/telecom-sms-dashboard.controller.js

```diff
@@ -231,4 +231,21 @@ export default class {
         this.loadingStats = false;
       });
   }
+
+  goToCreditTransfer() {
+    this.trackClick('report::transfer-credit');
+    return this.goToCreditTransfer();
+  }
+
+  goToCreditOrder() {
+    this.trackClick('report::add-credit');
+    return this.goToCreditOrder();
+  }
```

Comment:
**nitpick:** Could you please rename the functions in controller?
The same names are received as an input to the component and they are redefined in the controller which will cause confusion.

Reply:
You right, it's an error of copy past.
I fixed it.